api/external/exports.html

```diff
@@ -265,6 +265,12 @@ <h1>Exports<a class="headerlink" href="#exports" title="Permalink to this headli
         <li><strong>sms</strong> (<em>dictionary</em>) &#8211; Describes what should be exported from SMS related fields. </li>
         <ul class="first simple">
           <li><strong>fields</strong> (<em>array of strings</em>) &#8211; SMS fields to be exported.</li>
```

Comment:
Maybe something here like...


```suggestion
          <li><strong>fields</strong> (<em>array of strings</em>) &#8211; SMS fields to be exported, can be any or all of the example fields below.</li>
```